src/services/itemMembership/test/index.test.ts

```diff
@@ -63,16 +69,17 @@ describe('Membership routes tests', () => {
     });
 
     describe('Signed In', () => {
-      beforeEach(async () => {
-        ({ app, actor } = await build());
-      });
       it('Returns successfully for one id', async () => {
-        const { item, itemMembership } = await testUtils.saveItemAndMembership({ member: actor });
-
-        const member = await saveMember();
-        const membership = await testUtils.saveMembership({ item, account: member });
-
-        const memberships = [itemMembership, membership];
+        const {
+          actor,
+          items: [item],
+          itemMemberships,
+        } = await seedFromJson({
+          items: [{ memberships: [{ account: 'actor' }, { account: { name: 'bob' } }] }],
```

Comment:
Would it be possible to have 
```suggestion
          items: [{ memberships: [{ account: 'actor' }, { account: 'bob' }] }],
```

or even: 
```suggestion
          items: [{ memberships: [ 'actor', 'bob' ] }],
```

As it would be more concise.